index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="navContainer">
+            <div class="logo">
+                <img src="Sources/MyLogo.jpg" alt="Logo">
+            </div>
+            <ul class="navItems">
+                <li href="#">Portfolio</li>
+                <li href="#">About</li>
+                <li href="#">Contact</li>
+                <li href="#"><img src="Sources/Icons/Icon - Mail.svg" alt=""></li>
+            </ul>
+            <button type="button" class="hamburger">
+                <img src="Sources/Icons/Menu-Hamburguer.svg" alt="menu-hamburguer">
+            </button>
+        </div>
+    </header>
+    <section class="mainContainer">
+        <div class="mainContent">
+            <h1>Hey There.<br>I’m Jhonatan.</h1>
+            <h2>I'm a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="mainSocial">
+            <a href="https://github.com/JhonatanErosa">
+                <img src="Sources/Icons/GitHub_Icon.svg" alt="GitHub-Icon">
+            </a>
+            <a href="https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/">
+                <img src="Sources/Icons/Linkedin_Icon.svg" alt="Linkedin-Icon">
+            </a>
+            <a href="">
+                <img src="Sources/Icons/Hand_Icon.svg" alt="Hand-Icon">
+            </a>
+            <a href="https://twitter.com/erosa_c">
+                <img src="Sources/Icons/Twitter_Icon.svg" alt="Twitter-Icon">
+            </a>
+            <a href="">
+                <img src="Sources/Icons/Medio_Icon.svg" alt="Medio-Icon">
+            </a>
```

Comment:
Kindly, I suggest you make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 😄